app/views/spree/order_mailer/gift_card_email.html.erb

```diff
@@ -5,9 +5,11 @@
   You may want to print a copy of this message for easy reference later.
 </p>
 <table cellspacing="0" cellpadding="0" border="0" width="650">
-  <% if @gift_card.variant.images.present? %>
-    <tr><td colspan="2" style="padding-bottom: 15px"><%= image_tag @gift_card.variant.images.first.attachment.url(:product), style: 'width: 240px; height: auto' %></td></tr>
-  <% end%>
+  <%# if @gift_card.variant.images.present? %>
+    <%# <tr><td colspan="2" style="padding-bottom: 15px"> %>
+      <%#= image_tag url_for(@gift_card.variant.images.first.variant(resize: "100x100")) %>
```

Comment:
why we comment this?

Reply:
it gives error in line number 10 "variant is undefined" that's why i comment this block to run the email.

Reply:
revert these changes no need it